Tests/fixtures/smartphone-12.yml

```diff
@@ -297,8 +297,8 @@
     type: browser
     name: Aloha Browser
     version: "2.7.0.3"
-    engine: WebKit
-    engine_version: "537.36"
+    engine: Blink
+    engine_version: ""    
```

Comment:
end 4 spaces should be removed  'engine_version: ""
'

Reply:
Done.